### PR TITLE
Upd build tools 27

### PR DESCRIPTION
**Pull request by kesha-antonov**
#### What's this PR do?

Update build tools to 27

#### Which issue(s) is it related to?

Fixes https://github.com/tanguyantoine/react-native-music-control/issues/116

### CODE REVIEW

**Comment by Guichaguri** [comment]
I don't want to update the build tools version just yet. It can cause undesirable incompatibilities with other modules.

I'll leave this PR on hold, hopefully we'll be able to update it when React Native updates its API level. https://github.com/facebook/react-native/issues/17287

**Comment by kesha-antonov** [comment]
Which incompatibilities?

Wait for RN means wait forever.

**Comment by Guichaguri** [comment]
A lot of other modules might use build tools dependent libraries, such as other support lib packages or Google Play Services. We can not assume that they're going to be work fine. It's better to stick with what is stable than what isn't, mainly because this project is mostly fueled by PRs and not actual collaborators.

I understand your worries that React Native will take a long time to finally update it, but they will have to do it one way or another before august, at least that's guaranteed.

**Comment by kesha-antonov** [comment]
Ok! Got it.
I've updated many of modules and didn't see any problems. Although I didn't measure performance impact.

**Comment by tanguyantoine** [comment]
Since Google will require minimum version to be 26 it make sense. @Guichaguri what do you think ?

**Comment by jyotigupta007** [comment]
I have also faced the same issue as minimum version to be 26. So it make sense to merge this pull request and make it workable for other persons who are going to use this module in there projects. @Guichaguri  you should rethink on this.

**Comment by Guichaguri** [comment]
Yes, I think we're in a point where it's safe to update it to the API level 26+, React Native is in its way to updating to the newer API levels too. Can someone please test this PR?

**Comment by parveen-lukha** [comment]
Hi
As mentioned in my previous comment. "When I changed the files as Keha has done. It works fine for me."

**Comment by jyotigupta007** [comment]
@Guichaguri I have tested this pull request it is working fine for me. You can merge the same.

File: android/src/main/java/com/tanguyantoine/react/MusicControlModule.java [hunk 1]
```diff
@@ -107,8 +108,8 @@ public void init() {
         md = new MediaMetadataCompat.Builder();
         pb = new PlaybackStateCompat.Builder();
         pb.setActions(controls);
-        nb = new NotificationCompat.Builder(context);
-        nb.setStyle(new NotificationCompat.MediaStyle().setMediaSession(session.getSessionToken()));
+        nb = new android.support.v4.app.NotificationCompat.Builder(context);
```
Comment (Guichaguri) [inline android/src/main/java/com/tanguyantoine/react/MusicControlModule.java]: Can you use imports instead of declaring the package name directly?

**Comment by tanguyantoine** [comment]
@kesha-antonov  does that sounds doable ?

File: android/src/main/java/com/tanguyantoine/react/MusicControlModule.java (hunk 1 above)
Comment (kesha-antonov) [reply android/src/main/java/com/tanguyantoine/react/MusicControlModule.java]: There already imports

**Comment by kesha-antonov** [comment]
@tanguyantoine I can change code but don't have time to test it.
Only few weeks later

**Comment by tanguyantoine** [comment]
#157 does the trick

**Pull request closed without merging**